package.json

```diff
@@ -68,6 +68,7 @@
     "react-test-renderer": "~16.9.0",
     "scheduler": "^0.19.1",
     "ts-node": "^8.10.2",
+    "wdio-image-comparison-service": "^1.13.6",
```

Comment:
This dependency should be moved to PIW-Tools as now we are delivering screen shot testing for end-users

Reply:
True, pipeline didn't catch this unfortunately, because we don't do any screenshot testing yet in generated widget projects.

Reply:
Moved to PIW-Tools.